### PR TITLE
Updated docs to use latest config and follow same formatting

### PR DESCRIPTION
**Pull request by garethahealy**
#### What is this PR About?
The current docs in some places are out of date (i.e.: params that don't exist) and have differing format/styles. This PR updates everything to be current.

#### How do we test this?
Read the docs :)

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: vars/applier.txt [hunk 1]
```diff
@@ -1,114 +1,102 @@
...
-  userNames:
-  - system:serviceaccount:${NAMESPACE}:${SA_NAME}
-parameters:
-- description: The namespace to deploy into
-  name: NAMESPACE
-  required: true
-- description: The service account name
-  name: SA_NAME
-  required: true
+```java
```
Comment (InfoSec812) [inline vars/applier.txt]: Shoudn't the language tag be `groovy`?

File: vars/applyTemplate.txt [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
+
+### Requirements
+
+Requires Jenkins agent with:
+- OC
+
+### Parameters
+
+The method supports the following parameters:
+```java
```
Comment (InfoSec812) [inline vars/applyTemplate.txt]: Language tag should be `groovy`

File: vars/applyTemplate.txt [hunk 3]
```diff
@@ -1,30 +1,51 @@
 # applyTemplate
 
-This function processes template file with parameters file and then runs "oc apply" for the processing results.
-Also it has a workaround for the bug that if the DeploymentConfig already exists, it will break the Image Trigger.
+## Summary
```
Comment (InfoSec812) [inline vars/applyTemplate.txt]: Shouldn't this fille be `applyTemplate.md` instead of `.txt`?

Comment (garethahealy) [reply vars/applyTemplate.txt]: you'd of thought so, but no :)
- https://jenkins.io/doc/book/pipeline/shared-libraries/#directory-structure

File: vars/applyTemplate.txt (hunk 2 above)
Comment (garethahealy) [reply vars/applyTemplate.txt]: done

File: vars/applier.txt (hunk 1 above)
Comment (garethahealy) [reply vars/applier.txt]: done

**Comment by garethahealy** [comment]
Pushed requested changes

File: vars/applyTemplate.txt (hunk 3 above)
Comment (InfoSec812) [reply vars/applyTemplate.txt]: Weird!

**Pull request merged**